test/IRGen/objc_implementation.swift

```diff
@@ -47,6 +49,14 @@
   final var implProperty2: NSObject?
   
   @objc func mainMethod(_: Int32) { print(implProperty) }
+
+  @objc(copyWithZone:)
+  func copy(with zone: NSZone) -> Any {
```

Comment:
Would it make sense to also have an Interpreter test that exercises the emitted Obj-C protocol conformances, in addition to verifying the IR?

Reply:
I think verifying that the metadata is there will be enough.